# A pose node represents a pose that is used in the graph as a node.
std_msgs/Header header

int32 id
sensor_msgs/PointCloud2 scan
sensor_msgs/LaserScan laserscan
